funderfinder/sources/github_sponsors.py

```diff
@@ -167,7 +169,7 @@ def run(self, gh_project_slug: Union[str, None] = None) -> list:
         sources = []
         org = self.get_owner_name(gh_project_slug)
         num_org_funders = self.get_org_funder_count(org)
-        if org:
+        if num_org_funders:
```

Comment:
I think this may have been a typo, and that what we want here is whether there are > 0 org funders